util-images/access-tokens/cmd/main.go

```diff
@@ -0,0 +1,113 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	goflag "flag"
+	flag "github.com/spf13/pflag"
+	"io/ioutil"
+	"k8s.io/client-go/rest"
+	"net"
+	"os"
+	"path/filepath"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	certutil "k8s.io/client-go/util/cert"
+	"k8s.io/klog"
+)
+
+var (
+	namespace = flag.String("namespace", "", "Namespace the pod is running in")
```

Comment:
NIT: "the pod" is a bit ambiguous here. Maybe something more like "Namespace where the pod running this binary is running"?

Reply:
Changed to `Namespace that is targeted with request flow.` to describe exactly what is happening.